pyproject.toml

```diff
@@ -50,21 +45,22 @@ line-length = 120
 [tool.ruff.lint]
 # See complete list : https://docs.astral.sh/ruff/rules/
 select = [
-    "E",     # pycodestyle
-    "W",     # pycodestyle
-    "F",     # pyflakes
-    "I",     # isort
-    "UP",    # pyupgrade
-    "SIM",   # flake8-simplify
-    "YTT",   # flake8-2020
-    "FA",    # flake8-future-annotations
-    "ISC",   # flake8-implicit-str-concat
-    "PYI",   # flake8-pyi
-    "PT",    # flake8-pytest-style
-    "TID",   # flake8-tidy-imports
-    "ERA",   # eradicate
-]
-ignore = [
-  "E501",
-  "ISC001",
+  "E",   # pycodestyle
+  "W",   # pycodestyle
+  "F",   # pyflakes
+  "I",   # isort
+  "UP",  # pyupgrade
+  "SIM", # flake8-simplify
+  "YTT", # flake8-2020
+  "FA",  # flake8-future-annotations
+  "ISC", # flake8-implicit-str-concat
+  "PYI", # flake8-pyi
+  "PT",  # flake8-pytest-style
+  "TID", # flake8-tidy-imports
+  "ERA", # eradicate
 ]
+ignore = ["E501", "ISC001"]
+
+[[tool.mypy.overrides]]
```

Comment:
I'm not sure it's right thing to do, but I don't see any other alternative since pyudev does not want to provide type hints; we might as well do the block device enumeration ourselves (not that complicated).

Reply:
Since, we'll use pyudev to dump properties to, as discussed together today, we'll keep for now. Maybe we could contribute type hinting to pyudev.